src/org/thoughtcrime/securesms/ConversationListActivity.java

```diff
@@ -375,14 +374,9 @@ private void handleOpenpgp4fpr() {
         return;
       }
 
-      if (uri.getScheme().equalsIgnoreCase(OPENPGP4FPR)) {
-        String uriString = uri.toString().replaceFirst(OPENPGP4FPR, OPENPGP4FPR.toUpperCase());
+      if (uri.getScheme().equalsIgnoreCase(OPENPGP4FPR) || Util.isInviteURL(uri)) {
         QrCodeHandler qrCodeHandler = new QrCodeHandler(this);
-        qrCodeHandler.handleQrData(uriString);
-      } else if (INVITE_PAGE.equals(uri.getHost()) && uri.getEncodedFragment() != null) {
-        String data = OPENPGP4FPR.toUpperCase() + ":" + uri.getEncodedFragment().replaceFirst("&", "#");
-        QrCodeHandler qrCodeHandler = new QrCodeHandler(this);
-        qrCodeHandler.handleQrData(data);
```

Comment:
no need to parse invite links and convert them to QR data, core now supports handling links directly: https://github.com/deltachat/deltachat-core-rust/pull/5219

(needs a core update in the sub-module)

Reply:
i just updated core to recent `main` (as there is no release yet)